### PR TITLE
fix: avoid to have proxy arguments when calling the status bar entry command

### LINKED ISSUE #9378: Clicking on system tray button Version with available update does nothing and throws an error in dev console
### Bug description

Update E2E test is failing when trying to click with mouse cursor on the version button in the status bar with the exclamation mark icon where update is available. Nothing happens and so the test fails. It is suppose to show `Update Available Now` dialog. Instead, an error is thrown in the dev console.

Image

### Operating system

Windows 10

### Installation Method

Installer from website/GitHub releases

### Version

1.13.0

### Steps to reproduce

_No response_

### Relevant log output
```
An object could not be cloned.
Error: An object could not be cloned.
    at a (file:///D:/a/podman-desktop/podman-desktop/dist/win-unpacked/resources/app.asar/packages/renderer/dist/index-Bo55i8Ic.js:297:32891)
    at HTMLButtonElement.<anonymous> (file:///D:/a/podman-desktop/podman-desktop/dist/win-unpacked/resources/app.asar/packages/renderer/dist/index-Bo55i8Ic.js:297:33423)
    at HTMLButtonElement.s (file:///D:/a/podman-desktop/podman-desktop/dist/win-unpacked/resources/app.asar/packages/renderer/dist/index-Bo55i8Ic.js:2:14538)
```
### Additional context

_No response_

### PR DESCRIPTION
**Pull request by benoitf**
### What does this PR do?
some parameters can't be cloned but we're mostly interested in the values, not the object so grab values before calling the method

it fixes the update workflow clicking in the status bar entry

side effect from https://github.com/containers/podman-desktop/pull/9186

### How to test this PR?

fixes https://github.com/containers/podman-desktop/issues/9378

use issue to have steps

- [x] Tests are covering the bug fix or the new feature

### CODE REVIEW

**Review by deboer-tim — APPROVED** [review]
It's not clear from the description how this problem causes (or fixes) #9393, but I don't have any concerns with the change / code LGTM.

**Comment by benoitf** [comment]
I think I'm not 100% sure but I suppose that the usage of `$state()` is proxyfing the values of status bar entries (and then the command args)

**Comment by benoitf** [comment]
I'll merge the fix, if people disagree on the fix, I'll be please to sort it out in follow-up PRs but it seems it's blocking all our dependabot PR

**Comment by odockal** [comment]
@benoitf Tried locally and not everything works again. Thanks for the fix!

**Comment by benoitf** [comment]
not or now ? :-)

**Comment by odockal** [comment]
> not or now ? :-)

omg I was sure I updated the typo. Sry. NOW was it.

**Pull request merged**
